.github/workflows/myself.yaml

```diff
@@ -0,0 +1,38 @@
+name: Myself build environment
+
+# Build only on changes to the build environment itself
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - staging
+    paths:
+      - '.github/workflows/myself.yaml'
+      - 'buildenvs/myself.Dockefile'
+      - 'Makefile'
+
+jobs:
+  myself-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
To be removed, we should use our own registry :)

Reply:
I'll leave this in for now